### ISSUE TITLE
How to get the labels for the training and test sets

### ISSUE THREAD

**Issue by crush-kunkun** [Mingyue-Cheng/InstructTime#7]
Hi, I'm following your great work and trying to use the dataset provided in this project.

I can load the feature files (train.npy/test.npy) successfully, but I cannot find the corresponding training/test labels. Could you kindly tell me how to get or extract the labels for the training set?

Thanks a lot for your help!
